### PR TITLE
Add lightweight support for folia via MorePaperLib (not work)

### PR DESCRIPTION
**Pull request by mani1232**
I seem to have done everything right, but it doesn't work :(

### CODE REVIEW

**Comment by WiIIiam278** [comment]
You seem to have cloned the entire wiki for some reason, can you delete this?

File: common/src/main/java/net/william278/husktowns/visualizer/ParticleChunk.java [hunk 1]
```diff
@@ -29,8 +29,13 @@ public class ParticleChunk {
 
     private final Position[] positions = new Position[4];
 
+    Chunk chunk;
+    World world;
+
     private ParticleChunk(@NotNull Chunk chunk, @NotNull World world) {
         super();
+        this.chunk = chunk;
+        this.world = world;
```
Comment (WiIIiam278) [inline common/src/main/java/net/william278/husktowns/visualizer/ParticleChunk.java:38]: Why is this necessary?

**Review by WiIIiam278 — COMMENTED** [review]
I'd like to see the Task logic refactored to implement the new HuskHomes system introduced in 4.4, which works pretty well with folia as the state of a task is tracked within it.
https://github.com/WiIIiam278/HuskHomes2/blob/master/common/src/main/java/net/william278/huskhomes/util/Task.java
https://github.com/WiIIiam278/HuskHomes2/blob/master/bukkit/src/main/java/net/william278/huskhomes/util/BukkitTask.java

If you're up for doing this, I think we can get this merged soon :) -- we can disable advancements on folia if detected for now ig (unless they've been implemented recently).

File: common/src/main/java/net/william278/husktowns/visualizer/ParticleChunk.java (hunk 1 above)
Comment (mani1232) [reply common/src/main/java/net/william278/husktowns/visualizer/ParticleChunk.java:38]: Im need it for create PARTICLE in chunk by location

**Comment by mani1232** [comment]
> I'd like to see the Task logic refactored to implement the new HuskHomes system introduced in 4.4, which works pretty well with folia as the state of a task is tracked within it. https://github.com/WiIIiam278/HuskHomes2/blob/master/common/src/main/java/net/william278/huskhomes/util/Task.java https://github.com/WiIIiam278/HuskHomes2/blob/master/bukkit/src/main/java/net/william278/huskhomes/util/BukkitTask.java
> 
> If you're up for doing this, I think we can get this merged soon :) -- we can disable advancements on folia if detected for now ig (unless they've been implemented recently).

Ok I think I can do it soon

**Comment by WiIIiam278** [comment]
A heads up -- the license has now changed to Apache-2.0 (which is a more permissive open source license).

I hope that's all good with you :)

Thanks for your work.

**Comment by WiIIiam278** [comment]
Gonna close this as stale for now
